lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -190,6 +201,23 @@ def deactivate_services
         LinuxAdmin::Service.new("zookeeper").stop
         LinuxAdmin::Service.new("kafka").stop
       end
+
+      def assemble_ks_params
+        ks_params = {"-keystore" => keystore_path, "-validity" => 10_000, "-genkey" => nil, "-keyalg" => "RSA", "-storepass" => password, "-keypass" => password}
+
+        if server_host_is_ipaddr?
+          ks_alias = "localhost"
+          ks_params["-ext"] = "san=ip:#{server_host}"
+        else
+          ks_alias = server_host
+          ks_params["-ext"] = "san=dns:#{server_host}"
+        end
+
+        ks_params["-alias"] = ks_alias
+        ks_params["-dname"] = "cn=#{ks_alias}"
+
+        [ks_alias, ks_params]
```

Comment:
Can this return just the `ks_params` and have the callers of `ks_alias` be `ks_params["-alias"]`?

Reply:
Also, can `ks_params` be `keystore_params`?

Reply:
@bdunne 👍 
Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/149/commits/0da11790549b1b5a6f97370e6eca2c5198b36751